client/content_quota_report/directives/ContentQuotaReportView.js

```diff
@@ -0,0 +1,75 @@
+ContentQuotaReportView.$inject = ['contentQuotaReport', 'contentQuotaChart', '$interval', '$timeout'];
+
+/**
+ * @ngdoc directive
+ * @module superdesk.apps.analytics.content-quota-report
+ * @name sdContentQuotaReportView
+ * @requires contentQuotaReport
+ * @requires contentQuotaChart
+ * @requires $interval
+ * @requires $timeout
+ * @description A directive that displays the generated content quota report
+ */
+export function ContentQuotaReportView(contentQuotaReport, contentQuotaChart, $interval, $timeout) {
+    return {
+        template: require('../views/content-quota-report-view.html'),
+        scope: {},
+        link: function(scope, element, attrs, controller) {
+            var regenerateInterval = 60000,
+                interval = null,
+                chart = null;
+
+            /**
+             * @ngdoc method
+             * @name sdContentQuotaReportView#regenerateReport
+             * @description Regenerate the report and the chart
+             */
+            var regenerateReport = function() {
+                if (scope.contentQuotaReport) {
+                    contentQuotaReport.generate(scope.contentQuotaReport)
+                    .then((report) => {
+                        scope.contentQuotaReport = report;
+                        scope.generateChart();
+                    });
+                }
+            };
+
+            /**
+             * @ngdoc method
+             * @name sdContentQuotaReportView#resetInterval
+             * @description Reset the periodic generation of the chart
+             */
+            var resetInterval = function() {
+                if (angular.isDefined(interval)) {
+                    $interval.cancel(interval);
+                }
+                interval = $interval(regenerateReport, regenerateInterval);
```

Comment:
I modified regenerateReport function not to call generateChart and so, not to reset the interval anymore. The resetInterval function is called only when a new report is requested by the user.